.travis.yml

```diff
@@ -1,5 +1,9 @@
+dist: xenial
 language: node_js
 node_js:
-  - "9"
+  - "10"
   - "8"
   - "6"
+cache: yarn
+before_install:
+  yarn config set ignore-engines true
```

Comment:
What this for?

Reply:
yarn throws error when installing a transitive package where engine is marked as requiring node >=8
npm works fine. This command relaxes that check like npm.
https://stackoverflow.com/questions/45088031/how-to-ignore-incompatible-engine-node-error-on-installing-npm-dependencies-wi

Reply:
Which package?

Reply:
https://travis-ci.org/TrySound/rollup-plugin-size-snapshot/jobs/477973103
`ip-regex`